src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -759,7 +793,10 @@ private IUserSecurityStampStore<TUser> GetSecurityStore()
             CancellationToken cancellationToken = default(CancellationToken))
         {
             ThrowIfDisposed();
-            return await GenerateUserTokenAsync(user, Options.PasswordResetTokenProvider, "ResetPassword", cancellationToken);
+            var token = await GenerateUserTokenAsync(user, Options.PasswordResetTokenProvider, "ResetPassword", cancellationToken);
+            await LogResultAsync(IdentityResult.Success, user);
```

Comment:
I thought this one where we trace entering and existing a method was to be done as a separate thing across the framework components. Do we want to have that separately here ?